crates/bevy_render/src/render_ecs_resource.rs

```diff
@@ -0,0 +1,44 @@
+use std::marker::PhantomData;
```

Comment:
I think this file should be called `extract_resource.rs` (the `render_component.rs` was an artifact of the old RenderComponent trait)

Reply:
Maybe just rename `render_component.rs` to `extract_component.rs` while you're at it.

Reply:
Done.